### PR TITLE
Fixing EBS Raid creation and adding Encryption option to Raid

### PR DESCRIPTION
**Pull request by DrMerlin**
See AWS issue: https://github.com/opscode-cookbooks/aws/issues/129 and https://github.com/opscode-cookbooks/aws/issues/118.

This restores the ability to create EBS volumes as part of a Raid.  

It also adds the required passthrough to created encrypted volumes as a Raid.

### CODE REVIEW

**Comment by scalp42** [comment]
LGTM :+1:

File: resources/ebs_raid.rb [hunk 1]
```diff
@@ -31,3 +33,5 @@
 attribute :disk_type,             kind_of: String, default: 'standard'
 attribute :disk_piops,            kind_of: Integer, default: 0
 attribute :existing_raid,         kind_of: [TrueClass, FalseClass]
+attribute :disk_encrypted,        kind_of: [TrueClass, FalseClass], default: false
+attribute :disk_kms_key_id,       kind_of: String
```
Comment (juliandunn) [inline resources/ebs_raid.rb]: Should the `:disk_kms_key_id` be marked as a required parameter?

Comment (scalp42) [inline resources/ebs_raid.rb]: Nope @juliandunn aws-sdk will default to one:

```
Default master key that protects my EBS volumes when no other key is defined
```

Comment (DrMerlin) [inline resources/ebs_raid.rb]: What he said.

**Comment by mreeves1** [comment]
+1 - This is currently biting us too.

**Comment by purgatorio** [comment]
+1 on this

**Comment by liquid-sky** [comment]
+1 Pls merge

**Comment by mkantor** [comment]
Is there anything I could do to expedite this? Until this pull request is merged we're stuck on the 2.3.0 release, which has its own set of issues (it barfs in cn-north-1 for example).

**Comment by scalp42** [comment]
This PR is really straightforward, would like to know also what can be done to speed it up please.

Thanks in advance!

**Comment by mkantor** [comment]
I sliced out the tiny changeset that unbreaks ebs_raid and opened #156 (just in case that helps get the fix merged faster, I don't mean to undermine @DrMerlin's excellent work here).

**Comment by DrMerlin** [comment]
Since #156 was merged in lieu of this PR, I have opened PR #165 with the remaining content.

**Pull request closed without merging**